SimTracker/TrackAssociatorProducers/plugins/QuickTrackAssociatorByHitsImpl.cc

```diff
@@ -800,7 +800,7 @@ double QuickTrackAssociatorByHitsImpl::weightedNumberOfTrackClusters(iter begin,
       weightedClusters += weight;
     }
   }
-  LogTrace("QuickTrackAssociatorByHitsImpl") << "  total weighted clusters: " << weightedClusters;
 
+  LogTrace("QuickTrackAssociatorByHitsImpl") << "  total weighted clusters: " << weightedClusters;
```

Comment:
is there a special character? the lines look the same to me.

Reply:
The line number changed, maybe an added empty line is being misinterpreted in the comparison. I'll remove it and it will hopefully disappear. I can't see any things of hidden characters.